internal/commands/status.go

```diff
@@ -17,9 +17,11 @@ func statusCmd(dockerCli command.Cli) *cobra.Command {
 	var opts credentialOptions
 
 	cmd := &cobra.Command{
-		Use:   "status <installation-name>",
-		Short: "Get the installation status. If the installation is a docker application, the status shows the stack services.",
-		Args:  cli.ExactArgs(1),
+		Use:     "status INSTALLATION_NAME",
+		Short:   "Get the installation status of an application",
+		Long:    "Get the installation status of an application. If the installation is a Docker Application, the status shows the stack services.",
+		Example: "$ docker app status myapp",
```

Comment:
s/myapp/my-installation ?

Reply:
I'll make install, upgrade, status, and uninstall all use `myinstallation`.